packages/projects-docs/pages/learn/editors/settings.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Editor Settings
+description: Overview of settings and configurations of the CodeSandbox editor.
+---
+
+import { Tabs, WrapContent } from '../../../../../shared-components/Tabs'
+
+# Settings
+
+<Tabs tabs={["Web", "Desktop"]}>
+    <WrapContent>
+        ## Web editor settings
+        Under the main menu you will find the editor settings. This view contains settings for the editor and repository, management tools for environment variables, and options to toggle experiments on and off.
+
+        ### Editor settings
+
+        #### Codeium code completion
+
+        Codeium code completion is turned on by default in most cases. If at any time you want to disable this feature, you can do so from this tab.
```

Comment:
Is this accurate? I thought all our AI features were opt-in.

Reply:
I just check the code on this.
It's on by default on free and on sandboxes

There was a lot of back and forth, but this was what we landed on when we introduced the limit for free and agreed to a free future use of Codeium